### ISSUE TITLE
Can't change color of text

### ISSUE THREAD

**Issue by twiffy** [photopea/photopea#5754]
Selecting a new foreground color doesn't allow text to be composed in that color, or change the color of existing selected text.

**Comment by camoody1** [comment]
So, you would like Photopea to work as Photoshop does in this instance? Is that what you're asking?

**Comment by twiffy** [comment]
Yes, exactly. That's the case with all the other components of the UI that I've used, so I interpreted the lack of that in this area a bug -- but if that isn't a bug, I guess please consider it a feature request.

**Comment by photopea** [comment]
I was not aware of this feature of Adobe Photoshop.

It is implemented now. So, in short, if there is no text layer selected, the foreground color and the text color are "linked together".

**Comment by MichalSZZ** [comment]
> I was not aware of this feature of Adobe Photoshop.
> 
> It is implemented now. So, in short, if there is no text layer selected, the foreground color and the text color are "linked together".

@photopea exactly the same way it works for the Shapes in Photoshop. Can you add this feature there too?

When I have the foreground color selected and want to draw the shape for the first time - the fill color should be as in the color picker foreground. When I am on the shape layer and I am drawing another shape - it makes a new one on the new layer in the color of the previous one. When I change the foreground color and make a new (empty) layer and draw another shape - it should be in the new color fill from the color picker foreground.
